### PR TITLE
Add new alert that fires if etcd backup metrics are missing for 12h.

### PR DESCRIPTION
**Pull request by whites11**
Towards: https://github.com/giantswarm/giantswarm/issues/28783

This PR adds a new alert that pages if etcd backup operator metrics are not present in a MC

### Checklist

- [x] Update CHANGELOG.md
- [ ] Add [Unit tests](https://github.com/giantswarm/prometheus-rules/#testing)
- [ ] Follow [Alert structure](https://github.com/giantswarm/prometheus-rules/#how-alerts-are-structured)
- [ ] Consider [creating a dashboard](https://docs.giantswarm.io/getting-started/observability/grafana/custom-dashboards/) ([guidelines](https://intranet.giantswarm.io/docs/product/ux/guidelines/dashboards/)) (if it does not exist already) to help oncallers monitor the status of the issue.
- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).

### CODE REVIEW

**Review by calvix — APPROVED** [review]

File: helm/prometheus-rules/templates/alerting-rules/etcdbackup.rules.yml
Comment (fiunchinho) [inline helm/prometheus-rules/templates/alerting-rules/etcdbackup.rules.yml:74]: It would be best to add a new section to the recipe, explaining that the metrics are exposed by the controller, so the oncaller needs to check the deployment status.

**Review by QuentinBisson — APPROVED** [review]

**Review by njuettner — APPROVED** [review]

File: helm/prometheus-rules/templates/alerting-rules/etcdbackup.rules.yml
Comment (whites11) [reply helm/prometheus-rules/templates/alerting-rules/etcdbackup.rules.yml:74]: https://github.com/giantswarm/giantswarm/pull/28825/files

**Review by fiunchinho — APPROVED** [review]

**Comment by whites11** [comment]
waiting for opsrecipe to be merged before I merge this one

**Pull request merged**
